### PR TITLE
feat: add Tourette syndrom lolifier

### LINKED ISSUE #9: Add Tourette lolifer

### PR DESCRIPTION
**Pull request by devwebpeanuts**
close #9

### CODE REVIEW

**Comment by lebris** [comment]
What do you think about using a class to provide bad words, instead of using a static method. 

With that it will be possible to translate Tourette in french, german, klingon, etc

**Comment by devwebpeanuts** [comment]
You're right.

**Comment by devwebpeanuts** [comment]
Done

File: lib/Lolifiers/BadWordsProviders/DefaultProvider.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Monolol\Lolifiers\BadWordsProviders;
+
+use Monolol\Lolifiers\BadWordsProvider;
```
Comment (Niktux) [inline lib/Lolifiers/BadWordsProviders/DefaultProvider.php:5]: swearword

Comment (devwebpeanuts) [inline lib/Lolifiers/BadWordsProviders/DefaultProvider.php:5]: Yep, you're right. It's more accurate. I'll change it to the next refactoring

**Pull request merged**